EFRepository/GenericRepository.cs

```diff
@@ -220,5 +220,13 @@ public int SaveChanges()
 
             return result;
         }
+
+        /// <summary>
+        /// Saves the changes.
+        /// </summary>        
+        public async void SaveChangesAsync()
+        {
+            var result = await this.DbContext.SaveChangesAsync();            
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Remove this useless assignment to local variable "result". [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonarqube.com/coding_rules#rule_key=csharpsquid%3AS1854)
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Remove this unused "result" local variable. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonarqube.com/coding_rules#rule_key=csharpsquid%3AS1481)